src/dataMap/metaManager/metaSchema.js

```diff
@@ -2300,18 +2301,24 @@ export default () => {
      *
      * @memberof Options#
      * @type {object|boolean}
-     * @default {syncLimit: 50}
+     * @default {true}
```

Comment:
Not an object?
```suggestion
     * @default true
```

Reply:
That was the outdated version - I corrected this in the next commit.
I want to keep it as it is, which means the default and backwards-compatible value is `undefined`. I know we discussed it, that we could use `true` - although when I reconsidered that, I feel it would be a breaking change. Otherwise, it will be the only plugin that you can enable by updating its settings to `undefined`.

Reply:
Sure 👍 I just noticed the curly brackets. Good point though